src/index.html

```diff
@@ -1,13 +1,79 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./index.css"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link rel="preconnect"
+    href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body class="page">
+    <div class="product-card page__product-card"
+      data-qa="card"
+    >
+      <img
+        class="product-card__pic"
+        src="./images/imac.jpeg"
+        alt="Моноблок APPLE A1419 iMac 27' Retina 5K (MNED2UA/A)"
+      >
+
+      <h2 class="product-card__name">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </h2>
+
+      <p class="product-card__id">
+        Код товара: 195434
+      </p>
+
+      <div class="stars stars--4">
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <p class="stars__reviews">
+          Отзывов: 5
+        </p>
+      </div>
+
+      <div class="product-card__price-block">
+        <p class="product-card__price-name">
+          Цена:
+        </p>
+
+        <p class="product-card__price-num">
+          69 999 грн
+        </p>
+      </div>
+
+      <button class="product-card__buy-button">
+        <a class="product-card__buy-link"
+          data-qa="hover"
+          href="https://www.moyo.ua/ua/monoblok_apple_a1419_imac_27_retina_5k_mned2ua_a/361498.html"
+          target="_blank"
+        >
+          Купить
+        </a>
+      </button>
```

Comment:
You could simply leave link, without button.
Remember, you should change classes.

```suggestion
      <a class="product-card__buy-link"
        data-qa="hover"
        href="https://www.moyo.ua/ua/monoblok_apple_a1419_imac_27_retina_5k_mned2ua_a/361498.html"
        target="_blank"
      >
        Купить
      </a>
```